Test/Unit/Backend/DataStoreTest.php

```diff
@@ -53,4 +53,23 @@ public function testGetHostAPIUserKeyReturnsNull() {
     public function testGetHostAPIUserUniqueIdReturnsNull() {
         $this->assertNull($this->dataStore->getHostAPIUserUniqueId());
     }
+
+    public function testGetZoneIdReturnsValue() {
+        $domain = "domain";
+        $key = DataStore::ZONE_ID_KEY.$domain;
+        $zoneId = "zoneId";
+
+        $this->mockMagentoAPI->method('getValue')->with($key)->willReturn($zoneId);
+        $response = $this->dataStore->getZoneId($domain);
+        $this->assertEquals($zoneId, $response);
+    }
+
+    public function testSetZoneIdSetsValue() {
+        $domain = "domain";
+        $key = DataStore::ZONE_ID_KEY.$domain;
+        $zoneId = "zoneId";
+
+        $this->mockMagentoAPI->expects($this->once())->method('setValue')->with($key,$zoneId);
+        $this->dataStore->setZoneId($domain,$zoneId);
```

Comment:
`expects($this->once())` acts as an assertion.  It will fail if the method `setValue()` isn't called at least once.